python/lsst/analysis/drp/calcFunctors.py

```diff
@@ -132,43 +132,79 @@ def __call__(self, df):
 
 
 class CalcE(MultiColumnAction):
-    """Calculate a complex value representation of the ellipticity
-
-    This is a shape measurement used for doing QA on the ellipticity
-    of the sources.
+    """Calculate a complex value representation of the ellipticity.
 
     The complex ellipticity is typically defined as
-    E = ((ixx - iyy) + 1j*(2*ixy))/(ixx + iyy) = |E|exp(i*2*theta).
-
-    For plotting purposes we might want to plot |E|*exp(i*theta).
-    If `halvePhaseAngle` config parameter is set to `True`, then
-    the returned quantity therefore corresponds to |E|*exp(i*theta)
-    """
+    E = |E|exp(i*2*theta) = ((ixx - iyy) + 1j*(2*ixy))/(ixx + iyy).
```

Comment:
Could you add a few words indicating that the `ixx`, `ixy`, and `iyy` are the second brightness moments? Also, should you use `Ixx` and so on (with a capitalized `I`) here and in the rest of the file as it is used in `ellipticityType` below, for consistency?  (and it might also help to differentiate from the imaginary unit).

Reply:
In single-visit source tables, the convention has been to call this `ixx`. But for documentation purposes, calling it `Ixx` etc. makes sense.